packages/x-date-pickers/src/YearPicker/PickersYear.tsx

```diff
@@ -96,29 +104,25 @@ const PickersYearButton = styled('button')<{
   },
 }));
 
-const noop = () => {};
 /**
  * @ignore - internal component.
  */
-export const PickersYear = React.forwardRef<HTMLButtonElement, YearProps>(function PickersYear(
-  props,
-  forwardedRef,
-) {
+const PickersYearRaw = (props: PickersYearProps) => {
   const {
     autoFocus,
     className,
     children,
     disabled,
-    onClick,
-    onKeyDown,
+    selected,
     value,
     tabIndex,
-    onFocus = noop,
-    onBlur = noop,
+    onClick,
+    onKeyDown,
+    onFocus,
+    onBlur,
     ...other
   } = props;
   const ref = React.useRef<HTMLButtonElement>(null);
-  const refHandle = useForkRef(ref, forwardedRef as React.Ref<HTMLButtonElement>);
```

Comment:
I never really get the point of using  `useForkRef` or not

Reply:
From what I understand, it allows to merge a ref received on the component and a ref created internally and pass the merged version to the DOM.

`useForkRef` then makes sure that both the received ref and the internal ref are kept up to date